cobalt/black_box_tests/black_box_cobalt_runner.py

```diff
@@ -103,17 +105,18 @@ def JSTestsSucceeded(self):
     # JavaScript test logic completion.
     self.PollUntilFound('[' + _TEST_STATUS_ELEMENT_NAME + ']')
     body_element = self.UniqueFind('body')
-    return body_element.get_attribute(
-        _TEST_STATUS_ELEMENT_NAME) == _JS_TEST_SUCCESS_MESSAGE
+    return (body_element.get_attribute(_TEST_STATUS_ELEMENT_NAME) ==
+            _JS_TEST_SUCCESS_MESSAGE)
```

Comment:
Avoid wrapping return statements in parens if not necessary: go/pystyle#parentheses